wandb/util.py

```diff
@@ -153,11 +157,12 @@ def sentry_exc(
 ) -> None:
     if error_reporting_enabled():
         if isinstance(exc, str):
-            capture_exception(Exception(exc))
+            sentry_hub.capture_exception(Exception(exc))  # type: ignore
         else:
-            capture_exception(exc)
-        if delay:
-            time.sleep(2)
+            sentry_hub.capture_exception(exc)  # type: ignore
+    if delay:
+        time.sleep(2)
```

Comment:
@raubitsj, @kptkin and I had a question about this magic. We understand what/why it does, but not all the implications of doing things this way (see, e.g. wandb/sdk/internal/internal.py:L168)